app/views/events/show.html.erb

```diff
@@ -36,6 +35,16 @@
     <%= @event.place.full_adress %>
   </p>
 
+<<<<<<< HEAD
```

Comment:
Here is next conflict which you didn't resolve. Compere with develop and choose the right version.